.github/workflows/pr_verification.yml

```diff
@@ -92,3 +92,26 @@ jobs:
         uses: gradle/gradle-build-action@v2
         with:
           arguments: test
+
+  bugzilla_check:
+    name: bugzilla reference check
+    runs-on: ubuntu-latest
+    steps:
+      - name: Mask secrets
+        shell: bash
+        run: |
+          echo "::add-mask::${{ secrets.BUGZILLA_API_TOKEN }}"
+          echo "::add-mask::${{ secrets.GITHUB_TOKEN }}"
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+      - uses: actions/checkout@v3
+      - name: Install dependencies
+        run: pip install Bugzilla requests
```

Comment:
if you install python from the distro repository, then install also `python3-requests` from there too, it should be more than enough

Reply:
Updated to install `python3-requests`  in the 'Install packages' step.